packages/vuetify/src/effects/color.ts

```diff
@@ -0,0 +1,83 @@
+import { computed, ComputedRef, isRef } from 'vue'
+
+export interface ColorProps {
+  color?: string
```

Comment:
I think we should make the props text-color and by-color. Right now color can mean either and it’s confusing.

Reply:
That would be separate interfaces for both, any since there are actually more color props maybe it should be generic?

https://www.typescriptlang.org/play/#code/C4TwDgpgBAwg9gGzgJwArLmAzgHnZgOQEMBbCLKCAD2AgDsATCrYZASzoHMA+KAXigBvAFBQxUANpgMYYmSgco+WaXIBdAPwAuKC3ZcA3MIC+w4QGM4dFlEtI0MigLwy55SjXpNdrDjwAUdijKWDrKbhQAPkquqlgSagA0UAwQAGZEAK4IwKFKRMjAbEQIOPD2IS6Ecdy8AoLGAJQ6AEoQlsgMVSpkWMlEdCB1vCLitojBjvxQAILIyEQgAHRsWHMLIIETDphYjVAa4xVTOhJBO9hqRqLiyBDAmch0R5O7S3cMmeYQ-v7Su8lzvs+CMbmMxHcHk8oAB5ABGACt2sAlkQsFg2Jw6H9HMlRuCCWdtmodPiCeSoKBIDoAMq+LiJMEU8GpDLZYA6VlZHLxc5JJkU0yCxpM4x44xQNFQNodLrhOL9QbcEWmCxWGxc9lOISU6jAcooHQAcjScDgRqgpks1mAUCsEGU03OIX8RtoNANyCNyU1PMaRmtNmAAHc4CEndsXRI3XrPd6oEa4Zw40kUulubl-WZA4gIEskJx-PblCKcwg8wX-CGw44RcJizIlu79dsjA3MEtTXAjNWQksk56e6G+12DEA

Reply:
I'm not sure that's needed (generic solution). Looking at our current instances of setTextColor and setBackgroundColor, most usage seems to be either directly with `this.color`, or a conditional/computed.

splitting color into text-color and bg-color seems big enough that it needs its own RFC for discussion

Reply:
Valid point @nekosaur , let's punt on this feedback.